fdbserver/workloads/HealthMetricsApi.actor.cpp

```diff
@@ -147,20 +147,20 @@ struct HealthMetricsApiWorkload : TestWorkload {
 			for (const auto& ss : healthMetrics.storageStats) {
 				auto storageStats = ss.second;
 				self->detailedWorstStorageQueue = std::max(self->detailedWorstStorageQueue, storageStats.storageQueue);
-				traceStorageQueue.detail(format("Storage/%s", ss.first.toString().c_str()), storageStats.storageQueue);
+				traceStorageQueue.detail(format("StorageS%s", ss.first.toString().c_str()), storageStats.storageQueue);
```

Comment:
I think the consensus in recent discussion was to use `"Storage_%s"`and `"Tlog_$s"` for detail names like this.

Reply:
It seems that underline is not allowed here, because the first letter of UID (which is `ss.first`) is not in the upper-case. So do you think hyphen `-` is okay as well?

Reply:
That's fine.  It appears that the validator enforces `_<upperCase>` and there are usages of that form.  We could relax the validator without affecting them, but that would also enable names that do not comply with our notion of name style to slip through later.